torchci/lib/bot/ciflowPushTrigger.ts

```diff
@@ -182,6 +203,13 @@ async function handleLabelEvent(context: Context<"pull_request.labeled">) {
     );
     return;
   }
+
+  if (valid_test_config_labels.includes(label)) {
```

Comment:
so we don't want to do anything if they add this label? Maybe add a TODO here to make it a bit more explicit

Reply:
Noted. Let me add this as a TODO. Theoretically, we can do the same for these new labels as what we do for `ciflow/trunk`, but it would require a major refactoring of the current PyTorch CI to support smaller workflow granularity (per test config)